src/test/fuzz/util.cpp

```diff
@@ -14,6 +14,19 @@
 
 #include <memory>
 
+std::vector<uint8_t> ConstructPubKeyBytes(FuzzedDataProvider& fuzzed_data_provider, Span<const uint8_t> byte_data, const bool compressed) noexcept
```

Comment:
I prefer `byte_data` because we are reading in an array of bytes and turning it into `pk_data` by adding a prefix byte to indicate compressed or uncompressed. At the moment we read it in, it is not `pk_data`. We turn it into `pk_data` (i.e. a byte array with the correct prefix), and then that is returned and can be serialized or turned into a `CPubKey` object.